x/chainlink/keeper/keeper.go

```diff
@@ -468,3 +471,48 @@ func (k Keeper) RequestNewRound(ctx sdk.Context, requestNewRound *types.MsgReque
 
 	return ctx.BlockHeight(), ctx.TxBytes(), nil
 }
+
+func (k Keeper) AddAccount(ctx sdk.Context, acc *types.MsgAccount) (int64, []byte) {
+	accStore := ctx.KVStore(k.accountStoreKey)
+
+	a := k.cdc.MustMarshalBinaryBare(acc)
+
+	accStore.Set(types.GetAccountKey(acc.GetSubmitter().String()), a)
+
+	return ctx.BlockHeight(), ctx.TxBytes()
+}
+
+func (k Keeper) EditAccount(ctx sdk.Context, acc *types.MsgEditAccount) (int64, []byte, error) {
+	accStore := ctx.KVStore(k.accountStoreKey)
+	accSubmitter := acc.Submitter.String()
+	accountBytes := accStore.Get(types.GetAccountKey(accSubmitter))
+
+	var account types.MsgAccount
+	k.cdc.MustUnmarshalBinaryBare(accountBytes, &account)
+
+	// submitters must match
+	if acc.Submitter.String() != account.Submitter.String() {
```

Comment:
better to compare by bytes instead of string

Reply:
also, this checking step should happen in anteHandler, anteHandler is able to do both stateless and stateful checking

Reply:
ok, will move over there!